src/app.py

```diff
@@ -1,42 +1,43 @@
 from flask import Flask, render_template, request, redirect, url_for
 from src.models.index_view_model import IndexViewModel
-from src.trello_items import TrelloClient
-from src.trello_config import TrelloConfig
+from src.mongo_config import MongoConfig
+from src.mongo_db_client import MongoClient
+
 
 def create_app():
     app = Flask(__name__)
-    trello_client = TrelloClient(TrelloConfig())
+    mongo_client = MongoClient(MongoConfig())
```

Comment:
Really minor, but naming this `mongo_client` ties the app into the implementation of your data store reasonably tighting; if this was more generic `item_storage` then a similar future change would limit this file to only caring about this one line/imports